numba/core/ir_utils.py

```diff
@@ -1693,8 +1769,15 @@ def compile_to_numba_ir(mk_func, glbls, typingctx=None, targetctx=None,
     # perform type inference if typingctx is available and update type
     # data structures typemap and calltypes
     if typingctx:
-        f_typemap, f_return_type, f_calltypes, _ = typed_passes.type_inference_stage(
-                typingctx, targetctx, f_ir, arg_typs, None)
+        f_typemap, f_return_type, f_calltypes, _ = (
+            typed_passes.type_inference_stage(
+                typingctx,
+                targetctx,
+                f_ir,
+                arg_typs,
+                None,
+            )
+        )
```

Comment:
Maybe create a temporary for the outputs and subsequently unpack? The braces around the function call seems strange.